adhocracy-plus/templates/header.html

```diff
@@ -40,6 +40,10 @@
                             {% url 'organisation-imprint' organisation_slug=ORGANISATION.slug as organisation_imprint_url %}
                             <a href="{{ organisation_imprint_url }}" class="organisation-header__nav-link"><span class="{% if request.path == organisation_imprint_url %}is-active{% endif %}">{% trans 'Imprint' %}</span></a>
                         </li>
+                        <li class="organisation-header__nav-item">
+                            {% url 'organisation-terms-of-use' organisation_slug=ORGANISATION.slug as organisation_terms_of_use_url %}
+                            <a href="{{ organisation_terms_of_use_url }}" class="organisation-header__nav-link"><span class="{% if request.path == organisation_terms_of_use_url %}is-active{% endif %}">{% trans 'Terms of use' %}</span></a>
+                        </li>
```

Comment:
This is actually just to test the new template until clear where it should be linked..

Reply:
Cool! I think @phillimorland already started with updating the footer, so you can talk to her about where it needs to go (or leave out the links and just put the urls in the description of the PR) and then put the links somewhere after merging.